Lauvel 1.1

Lauvel opened the pull request:
Pull request du dossier Lauvel, version 1.1 

Ajoute de nouveaux PNJ, suite quêtes permettant au joueur de visiter la mine et forêt !

Lauvel committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Lauvel

Lauvel committed:
Item dropé

Chair muté droper sur le gerrier muté

Lauvel committed:
Lauvel  1.0

Fichier Lauvel 1.0, début du dossier Lauvel avec l'urtilisation de
GitHub

Lauvel committed:
Lauvel 1.0 modif

rajoute de l'alchimie

Lauvel committed:
Lauvel 1.1

Rajoute de la convers du Forgeron et c'est différentes quêtes qui va
permettre de faire connaitre aux joueur la mine à la fin des quêtes en
fesant la livraison pour le Chef Mineur.

Suite de la 1.1 bientôt:
- Ajoute du Chef Mineur et des different mineur
- Ajoute  du Chef de Chantier et des ouvrier
- Ajoute du Chef Bucheron et des buchrons

Lauvel committed:
Lauvel 1.1 modi 01

Ajoute de du Chef Mineur et des mineur
(Moitier des quêtes)

Lauvel committed:
Lauvel 1.1 modif 2

-Ajoute du Chef de Chantier
-Ajoute du Menuisier
-Ajoute du Garde Forestier

Suite de Quêtes fini !
Bientôt Lauvel 1.2:
- Donjon

Lauvel committed:
Revert "Lauvel 1.1 modif 2"

This reverts commit 4c765cf5a180799a6ec45c8dcd817c35432e442e.

Lauvel committed:
Lauvel 1.1 Correction

J'ai eu l'erreur de clic sur le bouton qui faut pas :s
Bref:
- Ajoute:
Chef de chantier
Cjef de mineur
Mineur
Garde Forestier
Menuisier

Bientôt Lauvel 1.2:
- Début de qêtes pour passer Guerrier !
- Donjon( Epreuve pour passer Guerrier )

crashoz commented on BetonQuest/lauvel/objectives.yml:
```diff
@@ -1,6 +1,11 @@
+Cc_1: setblock AIR ???;???;???;Gaia
+Cc_2: setblock AIR ???;???;???;Gaia
+Cc_3: setblock AIR ???;???;???;Gaia
+Cc_4: setblock AIR ???;???;???;Gaia  
```
l'objectif *setblock* n'existe pas à ma connaissance? Ni les locations avec *???*
Ca me semble être une erreur ou alors ce n'est pas fini!

HollishKid replied:
Non, l' "event" setblock sous ce format existe, mais pas l'objectif.

Si tu veux que le joueur place un bloc le format de l'objectif est le suivant:
obj_buches: block LOG:2 -16 events:reward notify

(négatif pour casser, positif pour placer, ici avec "notify" pour avertir de la progression le joueur).

Lauvel replied:
Ah oui, zut j'ai mis ça dans objective. Et oui ??? c’est normal puisque j'ai toujours pas mes batiment nécessaire à cette quêtes.

crashoz commented on MythicMobs/Items/Chair_Mute.yml:
```diff
@@ -0,0 +1,10 @@
+Chair_Mute:
+  Id: ROTTEN_FLESH
+  Data: 0
+  Display: 'Chair_Muté'
```
Attention, il faut être cohérent entre MythicMobs et BetonQuest. Ici le nom doit être **'Chair Mutée'** ( sans _ )
(et je crois bien que muté s'accorde avec chair)

crashoz commented on BetonQuest/lauvel/items.yml:
```diff
@@ -1 +1,2 @@
-
+Chair_Mute: ROTTEN_FLESH data:0 name:Chair_Mute lore:Chair_humaine_d'une_personne_muté;§2Objet_de_Quête
```
Attention, il faut être cohérent entre MythicMobs et BetonQuest. Ici le nom doit être **Chair_Mutée** ( avec le _ et le é ).

Lauvel replied:
Ah ok je vais toute suite changer alors ^^

crashoz reviewed (changes requested):
Petite incohérence entre l'item BetonQuest et MythicMobs. Il faut que l'item soit **exactement** le même sinon il ne sera pas reconnu correctement par BetonQuest. Sinon très bien 👍

crashoz commented:
Je me demande même si ça marche de mettre un item MythicMob dans BetonQuest, qu'en pense @HollishKid ?

HollishKid commented:
@crashoz @Lauvel 
BetonQuest n'est pas expert dans la gestion d'objets en réalité. J'ai fais la demande à Coosh d'améliorer le système, chose qui se fait progressivement.
Il me semble le "unbreakable" a été ajouté, mais qu'on n'a toujours pas la possibilité de mettre des attributs génériques (qui se font très bien avec MythicMobs !) dans BetonQuest, par exemple.

Après, si on s'applique et que les items sont identiques, j'ai déjà testé avec succès un item généré par MM avec une condition BQ.
C'est donc à bien vérifier et tester IG ; mais tu peux le faire marcher =)

Lauvel committed:
Lauvel 1.2.1

Eh voilà, notre ami Mike est de retour en tant que source d'information
:)
Nous enmène dans de nouvelle aventure  ^^
Découverte du nouveau personnage Da She.
Qui est ce mystèrieux Da She ? Qu'elle est sont lien avec la Reine ?
La suite peut-être au 1.2.2  ;)

Lauvel committed:
Lauvel 1.2.1 modif

Rajoute du journal qui n'a pas était commit enssemnle !  Bizarre.

Lauvel committed:
Modif item

modification pour la correspendance avec les item bq et mm.

Lauvel committed:
Guilde des Aventurier 1.1

-Modif de la phrase de commencement
-Modif ou je supprime les "r1" dans certains ligne
-Ajoute des Rangs 1~9
-Ajoute des récompense 1~9
-Ajoute des phrase information d'Itsuki

Lauvel committed:
Lauvel 1.1

suite qui c'est décocher

Lauvel committed:
Merge branch 'master' into Lauvel
